src/include/packing_inline.h

```diff
@@ -113,7 +113,11 @@ __pack_name_next(WT_PACK_NAME *pn, WT_CONFIG_ITEM *name)
         WT_CLEAR(*name);
         name->str = pn->buf;
         name->len = strlen(pn->buf);
+#ifdef __cplusplus
```

Comment:
Required to build on C++ and C.

Reply:
I think this deserves a comment explaining why it's here. Something like "C++ treats access to nested data structures differently than C, so we need to refer to them differently depending on the compiler."

Reply:
Good point, that should've had a comment.

Reply:
7936a12